docs/vm_builds/scripts/authorized_keys

```diff
@@ -0,0 +1 @@
+# SSH Key
```

Comment:
I don't see a reason for the file the way it stands. Adding a random example key would help.